src/test/resources/features/refapp-2.x/12-condition/conditions.feature

```diff
@@ -17,27 +20,32 @@ Feature: Patient's Conditions Managment
       | active   |
       | inactive |
 
-  @dashboard
+  @selenium
+  @condition
   Scenario: Set first condition to inactive
     And User clicks on set inactive button
     Then System should move condition to inactive section
 
-  @dashboard
+  @selenium
+  @condition
   Scenario: Set first condition to active
     And User clicks on set active button
     Then System should move condition to active section
 
-  @dashboard
+  @selenium
+  @condition
   Scenario: Edit first active condition
     And User edits active
     Then System should edit all active adjustable fields
 
-  @dashboard
+  @selenium
+  @condition
   Scenario: Edit first inactive condition
     And User edits inactive
     Then System should edit all inactive adjustable fields
 
-  @dashboard
+  @selenium
+  @condition
   Scenario: Delete first condition
     And User clicks delete condition
     Then System should trash first condition
```

Comment:
isn't the feature file supposed to be

```js
Feature: Patient's Conditions Managment

  Background:
    Given User clicks on Conditions from Patient dashboard
    Given a user clicks on the active visits link from home page
    When a user selects a patient from the active patient list
    Then the system loads the Patient dashboard page
    When a user clicks on Conditions from Patient dashboard
    Then System loads Manage Conditions Page

  @selenium
  @condition
  Scenario Outline: Adding an active or inactive condition
    And User clicks on Add new condition
    Then System on Add New Condition Page
@@ -17,27 +20,32 @@ Feature: Patient's Conditions Managment
      | active   |
      | inactive |

 
     # User Story: Set first condition to inactive
    And User clicks on set inactive button
    Then System should move condition to inactive section

     #User Story: Set first condition to active
    And User clicks on set active button
    Then System should move condition to active section

     #User Story: Edit first active condition
    And User edits active
    Then System should edit all active adjustable fields

     #User Story: Edit first inactive condition
    And User edits inactive
    Then System should edit all inactive adjustable fields

    #User Story: Delete first condition
    And User clicks delete condition
    Then System should trash first condition

Reply:
looks okayy but when run it throws something similar to this -> https://pastebin.com/X25U1rse

Reply:
doesn't the log come as a result of what was mentioned [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/174#issuecomment-958849299)?

Reply:
i did as you suggested, deleted my local repo, the cloned again. ran the test and it was throwing the initial errors/build failures we are trying to solve. However when i inserted those changes , it not-only fails but affects even other workflows

Reply:
have you figured out the better approach to the workflow test?